src/prelude.rs

```diff
@@ -0,0 +1,24 @@
+pub use super::chains::{ChainId, SUPPORTED_CHAINS};
+pub use crate::{
+    constants::{Rounding, MAX_UINT256},
+    entities::{
+        base_currency::{BaseCurrency, CurrencyLike},
+        currency::CurrencyTrait,
+        ether::Ether,
+        fractions::{
+            currency_amount::CurrencyAmount,
+            fraction::{Fraction, FractionLike, FractionTrait},
+        },
```

Comment:
Missing `percent` and `price`.

Reply:
> Missing `percent` and `price`.

don't think they are used anywhere or depends on who make the next pull we can just add it. can I add you as a reviewer.

Reply:
Sure.